tests/regression/roles/sonic_vxlan/defaults/main.yml

```diff
@@ -87,35 +87,67 @@ tests:
         vrf_map:
           - vni: 102
             vrf: "{{vrf2}}"
-  - name: del_test_case_06
-    description: Delete VRF properties
-    state: deleted
+  - name: test_case_06
+    description: Override VXLAN properties
+    state: overridden
     input:
-      - name: vtep1
-        source_ip: 1.1.1.1
-        primary_ip: 2.2.2.2
+      - name: vtep2
+        source_ip: 3.3.3.3
+        primary_ip: 4.4.4.4
         evpn_nvo: nvo5
         vlan_map:
           - vni: 101
             vlan: "{{vlan1}}"
-          - vni: 102
+        vrf_map:
+          - vni: 101
+            vrf: "{{vrf1}}"
+  - name: test_case_07
+    description: Replace VXLAN properties
+    state: replaced
+    input:
+      - name: vtep2
+        source_ip: 5.5.5.5
+        primary_ip: 6.6.6.6
+        evpn_nvo: nvo6
+        vlan_map:
+          - vni: 101
             vlan: "{{vlan2}}"
+          - vni: 102
+            vlan: "{{vlan1}}"
         vrf_map:
           - vni: 101
+            vrf: "{{vrf2}}"
+          - vni: 102
             vrf: "{{vrf1}}"
```

Comment:
Can we make test case names unified as test_case_xx, not del_test_case_xx?

Reply:
Yes, done.